tools/generate-yaml.sh

```diff
@@ -11,7 +11,7 @@ if [[ -n ${GCC_ARM} ]] ; then
   export PATH=${GCC_ARM}:$PATH
 fi
 
-: ${FLAVOR:="t15;tx16s;pl18;nv14;pl18u;nb4p;x9d;x9dp2019;x9e;xlite;xlites;x7;tpro;t20;f16;gx12;st16;pa01;tx15"}
+: ${FLAVOR:="t15;tx16s;nv14;pl18;pl18u;nb4p;x9d;x9dp2019;x9e;xlite;xlites;x7;tpro;t20;f16;gx12;st16;pa01;tx15;t15pro"}
```

Comment:
Unless something has changed since #6119, pl18 needs to be left where it was.

Reply:
very odd, but making the change anyway